test/connect-backbone-to-react.test.js

```diff
@@ -151,19 +162,22 @@ describe('connectBackboneToReact', function() {
         assert.equal(getEventHandlersSetCount(model, wrapperInstance), 0);
       });
     });
+
```

Comment:
above on the test `updates properties when model and collections change`, can you add an assertion that ensures `'Banana'` is rendered?